python/marvin/__init__.py

```diff
@@ -99,6 +101,9 @@ def __init__(self):
         self.add_github_message = True
         self._allowed_releases = {}
 
+        # Allow DAP queries
+        self._allow_DAP_queries = False
```

Comment:
You changed this.  This doesn't allow for future turning on for the public and collab, right?  Does this mean that someone can set this value to True, and spaxel queries will be re-enabled?

Reply:
Yes, I think allowing collab or public independently won't work with this implementation unless you think otherwise, so I changed to a boolean. Technically yes, it can be changed, but it's a private attribute. Ultimately, since we are not removing the code, just disabling it, everything we make can be disabled by somebody that knows the code, but I don't find that very likely.

Reply:
It won't but I think we could always add `config.access == 'collab'` into the condition wherever we use `allow_DAP_queries` to enable it for the collaboration.  Since I'm making some changes here and there I just want to be on the same page as what you're thinking.

Reply:
Yeah, that makes sense. It means if somebody is able to hack it at least would need to be a member of the collaboration.